e2e-test/e2etest.py

```diff
@@ -0,0 +1,60 @@
+
+from helper import *
+import os
+
+# Setup configuration
+# These values should be injected into the environment before setup
+azure_token = os.environ.get("AZURE_TOKEN")
+bee_name = os.environ.get("BEE_NAME")
+billing_project_name = os.environ.get("BILLING_PROJECT_NAME")
+#TODO: Take in arguments from the command line?
+cbas=False
+wds_upload=True
+cbas_submit_workflow=False
+test_cloning=False
+
+def run_workspace_app_test(cbas, wds_upload, cbas_submit_workflow, test_cloning):
+
+    setup(bee_name)
+    header = {"Authorization": "Bearer " + azure_token};
+
+    # create the workspace
+    workspace_id, workspace_name = create_workspace(cbas, billing_project_name, header)
+
+    # track to see when the workspace WDS is ready to upload data into them
+    # sleep for 5 minutes to allow WDS to start up, if no wds, only sleep 2 minutes to let cbas start up
+    if wds_upload:
+        time.sleep(200)

```

Comment:
Might be worth printing something right before sleeping, and include the amount of time to be slept, so if you're actively watching this you can extrapolate when it will keep going.

Reply:
Good call, I had a print statement about sleeping somewhere at some point in some iteration but seems like it got left out.

Reply:
possible suggestion here to make this better - instead of sleeping actually pool for wds url/status in leo until you can get it (sometimes it may be faster then 200 seconds)